pkg/framework/aries/framework.go

```diff
@@ -23,9 +23,10 @@ type DIDResolver interface {
 
 // Aries provides access to clients being managed by the framework.
 type Aries struct {
-	transport     api.TransportProviderFactory
-	didResolver   DIDResolver
-	storeProvider storage.Provider
+	transport      api.TransportProviderFactory
+	didResolver    DIDResolver
+	storeProvider  storage.Provider
+	newProtocolSvc []api.NewProtocolSvc
```

Comment:
I'm finding this field name confusing. what is meant by "new"?

Reply:
(perhaps something like protocolSvcCreators to match the other comment).

Reply:
done